test/Microsoft.OpenApi.Readers.Tests/V31Tests/ReferenceSamples/rootComponentSchemaReference.yaml

```diff
@@ -0,0 +1,24 @@
+openapi: 3.1.0
+info:
+    title: Reference at the root of a component schema
+    version: 1.0.0
+paths:
+    /items:
+        get:
+          responses:
+            200:
+              description: ok
+              content:
+                application/json:
+                  schema:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/specialitem'
+components:
+    schemas:
+      specialitem:  # Use the item type but provide a different title for the type
+        title: Special Item
```

Comment:
sanity check here, I thought it was only description and summary that could be overridden?

Reply:
Ah yes, getting rid of the comment as the title is not meant to be overridden

Reply:
should the title even be present then? what is it meant to validate?